youtube_dl/extractor/zdf.py

```diff
@@ -39,10 +39,22 @@ def _extract_player(self, webpage, video_id, fatal=True):
 
 
 class ZDFIE(ZDFBaseIE):
-    _VALID_URL = r'https?://www\.zdf\.de/(?:[^/]+/)*(?P<id>[^/?]+)\.html'
+    IE_NAME = "ZDF-3sat"
```

Comment:
A bit confusing, since the filename is still zdf.py

Reply:
Should I rename the file?